tests/integration/maven_artifact/test_downloader.py

```diff
@@ -1,11 +1,12 @@
 import os
 import tempfile
 
-from maven_artifact import Artifact, Downloader
+from maven_artifact import Downloader
+from maven_artifact.utils import Utils
 
 
 def test_downloader_of_existing_artifact():
-    artifact = Artifact.parse("org.apache.solr:solr:war:3.5.0")
```

Comment:
any particular reason why this was moved?

Reply:
I was perhaps a bit speedy there (and I was doing this at 3am if that's an excuse). I can revert it.

Reply:
That said, it would perhaps be useful with method name that explains what it parses from?

Reply:
Reverted to Artifact.parse.